iroha/src/block.rs

```diff
@@ -290,6 +297,22 @@ pub struct ValidBlock {
 }
 
 impl ValidBlock {
+    /// # Errors
+    /// Asserts specific instruction number of instruction constraint
+    pub fn check_instruction_len(&self, max_instruction_len: usize) -> Result<()> {
```

Comment:
To have consistent and descriptive naming - please change function name to `check_instructions_length`

Reply:
There is same named method `check_instruction_len` for transactions. Should we change it everywhere?

Reply:
`len` is okay to my mind. It is a Rust naming convention across std lib and many other crates, so in our codebase we also use it.

Reply:
This is mentioned in our [styleguide](https://github.com/hyperledger/iroha/blob/iroha2-dev/CONTRIBUTING.md#rust-style-guide)